resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentCredentialServiceImpl.java

```diff
@@ -78,6 +79,7 @@ public class ResidentCredentialServiceImpl implements ResidentCredentialService
 	private static final String PARTNER_TYPE = "partnerType";
 	private static final String ORGANIZATION_NAME = "organizationName";
 	private static final String INDIVIDUAL_ID = "individualId";
+	private static final StringBuilder DATA = new StringBuilder("data");
```

Comment:
Why  Stringbuilder is needed as constant?

Reply:
Changed it to normal string.